values-schema.yaml

```diff
@@ -1525,7 +1525,7 @@ properties:
         patternProperties:
           ^[a-z-]+$:
             $ref: '#/definitions/team'
-        type: object
+        type: array
```

Comment:
This makes more sense to me syntactically, but I don't know what it's gonna break.

<img width="164" alt="Screenshot 2020-12-11 at 13 16 53" src="https://user-images.githubusercontent.com/20700197/101902564-28863180-3bb3-11eb-96cf-16467a928e43.png">

Reply:
Either that or "teams" should just be one team of course.